### DISCUSSION TITLE
Copilot is failing to "cooperate" every few lines, and crashes

### DISCUSSION THREAD

**Discussion by 6vz** [community/community#43831] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Sooo... Today i Noticed that Copilot is not working properly on all my devices (Mac Mini M1 with VSCode & IntelliJ, and Thinkpad T470 with VSCode). Every 2-3 lines it keeps doing that (see image), and i have to tab in'n'out ~10 times to work. Can any Copilot employee look into that? I found it unusual and strange<br>
![image](https://user-images.githubusercontent.com/92748380/211547917-0b6f3566-4c8b-4e87-93df-1da5c7a50e4d.png)<br>
If anyone experiences same problems, and know how to fix them, please lmk :)<br>Thanks

**Comment by DuckyBlender** [comment]
Same here

**Comment by iamcjwilliams** [comment]
@6vz Sorry for the trouble here, Could you provide a few more details on how this can be replicated?

**Reply by iamcjwilliams** [reply]
@wojtekxtx Have you completed the troubleshooting steps mentioned here: Sorry for the trouble here, please visit this doc to troubleshoot your issue: https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting

**Comment by 6vz** [comment]
@iamcjwilliams 
Hi! There is no specific way to recreate that, it happens randomly.
All my other people & friends have the same issue. 

Copilot is passing ratelimits for it

**Reply by iamcjwilliams** [reply]
@6vz please refer to this post: https://github.com/community/community/discussions/43987
